docs/proposals/rollout/rollout.md

```diff
@@ -154,11 +154,18 @@ type ApplicationSyncPolicy struct {
 
 type RolloutConfig struct {
     // Testloader defines whether to install testloader for users. Default is true.
-    // Testloader generates traffic during canary analysis.
+    // Testloader generates traffic during rollout analysis.
     // If set it to false, user need to install the testloader himself.
     // If set it to true or leave it blank, Kurator will install the flagger's testloader.
     // +optional
-    TestLoader bool `json:"testLoader,omitempty"`
+    TestLoader *bool `json:"testLoader,omitempty"`
+
+    // removeTestloaderOnDelete defines wether to remove Testloader when delete 
+    // Application.Spec.ApplicationSyncPolicy.Rollout.
+    // If set true, Kurator will remove Testloader when delete 
+    // Application.Spec.ApplicationSyncPolicy.Rollout.
+    // Defines to false.
+    RemoveTestloaderOnDelete *bool `json:"removeTestloaderOnDelete,omitempty"`
```

Comment:
assume this can be set simultaneously with TestLoader. 

How about adding a struct for testLoader
```golang
type TestLoader struct{
  Enable bool
  RemoveOnDelete bool
}
```